crates/orchestrator/src/queue/job_queue.rs

```diff
@@ -42,12 +43,17 @@ pub struct JobQueueMessage {
     pub id: Uuid,
 }
 
-#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone, Display)]
 pub enum WorkerTriggerType {
+    #[strum(serialize = "Snos")]
     Snos,
+    #[strum(serialize = "Proving")]
     Proving,
+    #[strum(serialize = "ProofRegistration")]
     ProofRegistration,
+    #[strum(serialize = "DataSubmission")]
     DataSubmission,
+    #[strum(serialize = "UpdateState")]
     UpdateState,
 }
```

Comment:
```suggestion
#[derive(Debug, Serialize, Deserialize, PartialEq, Clone, Display)]
#[strum(serialize_all = "PascalCase")]
pub enum WorkerTriggerType {
    Snos,
    Proving,
    ProofRegistration,
    DataSubmission,
    UpdateState,
}
```

Reply:
Interesting, was unaware about this, Thanks!!!